about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="./images/favicon.ico" type="image/x-icon" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="about.css" />
+    <title>About</title>
+  </head>
+  <body>
+    <header>
+      <div id="first-header">
+        <img src="./images/logo-facebook.png" alt="" />
+        <img src="./images/logo-tweeter.png" alt="" />
+        <ul id="navbar-h1">
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+    </header>
+    <div id="second-header">
+      <a href="./index.html" id="logo-header"
+        ><img src="./images/logo.png" alt=""
+      /></a>
+      <ul id="navbar-h2">
+        <li><a href="./about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li><button>CC Campaign</button></li>
+      </ul>
+    </div>

```

Comment:
You did a great job working on this project! 💯 However, there's still room to improve your HTML semantics!

- [ ] I think it is better to include the second header inside your `<header>` tag as both toolbars compose the heading part of your page. 🔧 
- [ ] I suggest that you wrap your `<ul>` containing your navigation links inside a `<nav>` tag. This will indicate to the browser that these links' role is navigating your page. 🔧 


⚠️ **Consider applying these changes to your `index.html` too!** ⚠️